src/Orckestra.Composer.Search/Api/SearchController.cs

```diff
@@ -158,6 +159,40 @@ public virtual async Task<IHttpActionResult> GetSearchResults(GetSearchResultsRe
             return Ok(viewModel);
         }
 
+        [ActionName("searchBySkus")]
+        [HttpPost]
+        [ValidateModelState]
+        public virtual async Task<IHttpActionResult> GetSearchResultsBySkus(GetSearchResultsBySkusRequest request)
+        {
+            var Skus = request.Skus;
+            var queryString = HttpUtility.ParseQueryString(request.QueryString ?? "");
+            var SelectedFacets = SearchUrlProvider.BuildSelectedFacets(queryString).ToList();
+            var Keywords = queryString[SearchRequestParams.Keywords];
+            var BaseUrl = RequestUtils.GetBaseUrl(Request).ToString();
+            BaseSearchViewModel viewModel;
+
+            var searchCriteria = await BaseSearchCriteriaProvider.GetSearchCriteriaAsync(Keywords, BaseUrl, true).ConfigureAwait(false);
```

Comment:
add also IncludeFacets to GetSearchResultsBySkusRequest   parameter and use it in searchCriteria